incubator/drone/values.yaml

```diff
@@ -0,0 +1,117 @@
+## Drone
+## ref: https://hub.docker.com/r/drone/drone
+##
+image: drone/drone:0.5
```

Comment:
it looks like drone 0.6 was just released a few hours ago? https://hub.docker.com/r/drone/drone/tags/